tests/console/system_prepare.pm

```diff
@@ -83,6 +83,10 @@ sub run {
             services::registered_addons::full_registered_check;
         }
     }
+
+    if (get_var('COLLECT_COREDUMPS') && script_run('which coredumpctl') != 0) {
+        zypper_call 'in systemd-coredump';
+    }
```

Comment:
I think here it's easier and more efficient to avoid the change back to perl scope and back into the SUT:

```suggestion
    assert_script_run 'rpm -q systemd-coredump || zypper -n in systemd-coredump' if get_var('COLLECT_COREDUMPS');
```

Reply:
done